phpstan.neon.dist

```diff
@@ -8,3 +8,6 @@ parameters:
 		- %rootDir%/../../../vendor/*
 
 		- %rootDir%/../../../src/DependencyInjection/Configuration.php
+
+		- %rootDir%/../../../src/Drivers/*
```

Comment:
What's wrong with them?

Reply:
`
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  Line   src/Drivers/DatabaseDriver.php
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  58     Property Lexik\Bundle\MaintenanceBundle\Drivers\DatabaseDriver::$pdoDriver (Lexik\Bundle\MaintenanceBundle\Drivers\PdoDriver) does not accept
         Lexik\Bundle\MaintenanceBundle\Drivers\Query\DsnQuery.
  61     Parameter #1 $em of class Sonata\HelpersBundle\Drivers\Query\DefaultQuery constructor expects Doctrine\ORM\EntityManager, Doctrine\Common\Persistence\ObjectManager given.
  61     Property Lexik\Bundle\MaintenanceBundle\Drivers\DatabaseDriver::$pdoDriver (Lexik\Bundle\MaintenanceBundle\Drivers\PdoDriver) does not accept Sonata\HelpersBundle\Drivers\Query\DefaultQuery.
  63     Parameter #1 $em of class Sonata\HelpersBundle\Drivers\Query\DefaultQuery constructor expects Doctrine\ORM\EntityManager, Doctrine\Common\Persistence\ObjectManager given.
  63     Property Lexik\Bundle\MaintenanceBundle\Drivers\DatabaseDriver::$pdoDriver (Lexik\Bundle\MaintenanceBundle\Drivers\PdoDriver) does not accept Sonata\HelpersBundle\Drivers\Query\DefaultQuery.
  73     Call to method setSiteId() on an unknown class Lexik\Bundle\MaintenanceBundle\Drivers\PdoDriver.
  83     Call to method setSiteId() on an unknown class Lexik\Bundle\MaintenanceBundle\Drivers\PdoDriver.
  93     Call to method setSiteId() on an unknown class Lexik\Bundle\MaintenanceBundle\Drivers\PdoDriver.
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------

 ------ ---------------------------------------------------------------------------------------------------------------------
  Line   src/Drivers/Query/DefaultQuery.php
 ------ ---------------------------------------------------------------------------------------------------------------------
  50     Property Lexik\Bundle\MaintenanceBundle\Drivers\Query\PdoQuery::$db (PDO) does not accept Doctrine\DBAL\Connection.
  79     Method Sonata\HelpersBundle\Drivers\Query\DefaultQuery::selectQuery() should return array but returns null.
 ------ ---------------------------------------------------------------------------------------------------------------------

 ------ --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  Line   tests/Drivers/Query/DefaultQueryTest.php
 ------ --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  48     Call to method PHPUnit\Framework\Assert::assertSame() with Doctrine\DBAL\Connection&PHPUnit\Framework\MockObject\MockObject and PDO will always evaluate to false.
  79     Parameter #1 $db of method Sonata\HelpersBundle\Drivers\Query\DefaultQuery::deleteQuery() expects PDO, Doctrine\DBAL\Connection&PHPUnit\Framework\MockObject\MockObject given.
  109    Parameter #1 $db of method Sonata\HelpersBundle\Drivers\Query\DefaultQuery::selectQuery() expects PDO, Doctrine\DBAL\Connection&PHPUnit\Framework\MockObject\MockObject given.
  153    Parameter #1 $ttl of method Sonata\HelpersBundle\Drivers\Query\DefaultQuery::insertQuery() expects int, string given.
  153    Parameter #2 $db of method Sonata\HelpersBundle\Drivers\Query\DefaultQuery::insertQuery() expects PDO, Doctrine\DBAL\Connection&PHPUnit\Framework\MockObject\MockObject given.
`